Bottom: ee1ec0671c4289dc8ab224bd5012b7666a8d7b0b
Top:    34068b697a35f8c06872f0bd306517136eea2b2b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 23:56:11 +0000

Copy contour is_round information when copying polygons


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 364d472..039b9b5 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -3478,6 +3478,10 @@ poly_CopyContour (PLINE ** dst, PLINE * src)
   (*dst)->xmin = src->xmin, (*dst)->xmax = src->xmax;
   (*dst)->ymin = src->ymin, (*dst)->ymax = src->ymax;
   (*dst)->area = src->area;
+  (*dst)->is_round = src->is_round;
+  (*dst)->cx = src->cx;
+  (*dst)->cy = src->cy;
+  (*dst)->radius = src->radius;
 
   for (cur = NEXT_EDGE (&src->head); cur != &src->head; cur = NEXT_VERTEX (cur))
     {
